src/main/java/com/google/devtools/build/lib/bazel/repository/starlark/StarlarkRepositoryContext.java

```diff
@@ -361,7 +372,7 @@ public String readFile(Object path, StarlarkThread thread)
   }
 
   // Create parent directories for the given path
```

Comment:
Interesting, we didn't throw this exception before, how could the code compile successfully?

Reply:
Because `java.nio.file.InvalidPathException` is an unchecked exception.

Reply:
Ah ha, thanks!